dependencies.gradle

```
@@ -9,27 +9,43 @@ project.ext {
                     minSdk    : 21,
                     java      : '1.8',
             ],
-            android : [
-                    lifecycleVersion: "2.2.0",
+            koin    : [
+                    koinVersion: "2.1.5"
```

It's better to revert back the `android` group name. Also, I think `Version` suffix is redundant.
```suggestion
            android    : [
                    koin: "2.1.5"
```

---

Done, could you check that again?